vulns.go

```diff
@@ -334,6 +344,12 @@ func getSnykCodeIssueWithoutTickets(flags flags, projectID string, tickets map[s
 						}
 						json.Unmarshal(bytes, &issueDetail)
 
+						// checking if the issue is ignored
+						if e.K("attributes").K("ignored").Bool().Value == true {
+							fmt.Println("skipping")
```

Comment:
do we need to print anything? Do we do this for other issue types? If yes lets improve the message to explain what is being skipped "Vuln VULN_ID is ignored, skipping"